app/controllers/stories_controller.rb

```diff
@@ -85,12 +87,25 @@ def import
   end
 
   def export
-    csv = CSV.generate(headers: true) { |csv|
-      csv << CSV_HEADERS
-      @project.stories.by_position.each do |story|
-        csv << story.attributes.slice(*CSV_HEADERS)
+    csv = if params[:export_with_comments] == "1"
+      CSV.generate(headers: true) do |csv|
+        csv << CSV_HEADERS.append("comment")
+        comments = []
+        @project.stories.includes(:comments).by_position.each do |story|
```

Comment:
there's an issue here, every new story in the loop export all the comments of the previous stories too

Note the second story has the comment for the first story too
![image](https://github.com/fastruby/points/assets/188464/1a33787e-f1a4-4f42-ad3d-335efa83d9f4)